config/environments/development.rb

```diff
@@ -25,7 +25,7 @@
   config.assets.debug = false
   config.assets.compress = false
 
-  config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.delivery_method = :letter_opener if ENV['LETTER_OPENER'] == 'true'
```

Comment:
how about:

```
config.action_mailer.delivery_method = :letter_opener unless ENV['LETTER_OPENER'] == 'false'
```

Reply:
oh yeah you said it would be easier in production that way. I will change it.